inference/core/cache/model_artifacts.py

```diff
@@ -105,11 +108,73 @@ def get_cache_file_path(file: str, model_id: Optional[str] = None) -> str:
     return os.path.join(cache_dir, file)
 
 
+def _rmtree_onerror(func, path, exc_info):
+    """Error handler for shutil.rmtree."""
+    import errno
```

Comment:
Why is this import in the function definition?

Reply:
Good observation, moved it to the top.